### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_EditTags_Keyboard:Unable to Edit tags page with the help of keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3940]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
4.Navigate to any file present under Files tab-->Hit enter-->View Menu-->Cell Toolbar-->Tags. Click on Edit Tags button appeared on page.
5.Use keyboard to navigate to page with the help of any of the keyboard keys.
**Actual:**
Unable to navigate to the page according to repro steps with the help of keyboard.
Expected:
We should be able to navigate to the page with the help of keybaord.
 
**UserImpact:**
User will not able to navigate to the page and do unnecessary traversing if focus does not remain intact within the pop-up.
 **Attachment**:

[A11y_AzureNotebooks_Jupyter_EditTags_Keyboard1.pptx](https://github.com/jupyter/notebook/files/2397081/A11y_AzureNotebooks_Jupyter_EditTags_Keyboard1.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
